src/main/java/io/citrine/jcc/search/pif/result/PifSearchResult.java

```diff
@@ -116,5 +136,8 @@ private void readObject(ObjectInputStream in) throws IOException, ClassNotFoundE
      */
     private void readObjectNoData() throws ObjectStreamException {}
 
+    /** The time that the record was updated. */
+    private String updatedAt;
```

Comment:
Since this is a datetime stored as a string, I'd appreciate a comment about the format of the date. I know this is a late place to bring this up, sorry for not catching it earlier.

Reply:
The format is ISO-8061 (ISO-OFFSET-DATE-TIME), for example: 2017-01-01T00:00:00.001-08:00

Reply:
It's more useful to have this information in the docstring, but if this is consistent with the other updatedAt fields elsewhere in PifSearchResult it's fine.